### PR TITLE
The "Autosave" button would look better if centered - JIRA 10786

### PR DESCRIPTION
**Pull request by hvrana**
I tested this on inspect developer tool of browser. It looks much better after few changes

### CODE REVIEW

File: xwiki-platform-core/xwiki-platform-web/src/main/webapp/resources/js/xwiki/editors/autosave.css [hunk 1]
```diff
@@ -20,4 +20,4 @@
 #body #autosaveControl label.frequency, #body #autosaveControl label.frequency span {
   text-transform: none;
   font-weight: normal;
-}
```
Comment (tmortagne) [inline xwiki-platform-core/xwiki-platform-web/src/main/webapp/resources/js/xwiki/editors/autosave.css]: Better keep the newline at the end of the file.

Comment (hvrana) [reply xwiki-platform-core/xwiki-platform-web/src/main/webapp/resources/js/xwiki/editors/autosave.css]: sorry, I didn't get it. Do you mean a blank new line at the end?

**Comment by evalica** [comment]
You should also provide a screenshot with your change, since I doubt your changes totally fix the problem. 
An important aspect of this task is the cross browser aspect. So make sure you test you changes on all supported browsers, see http://dev.xwiki.org/xwiki/bin/view/Community/BrowserSupportStrategy
Vertical-alignment is especially problematic for IE.

**Comment by evalica** [comment]
Issue already fixed by https://jira.xwiki.org/browse/XWIKI-14162 . Also https://jira.xwiki.org/browse/XWIKI-10786 is marked as "Cannot reproduce".

**Comment by evalica** [comment]
Thank you for your PR.

**Pull request closed without merging**